server/API/REST.php

```diff
@@ -552,12 +559,13 @@ function book_lecture($vars) {
 			}
 
 			// Success
+			$in_wait_list = check_user_in_waiting_list($lectureId);
 			//I send a confirmazion email
 			$mail_result = mail($user_data["email"], "Confirmation of " . $lecture["name"] . " lecture booking", "You did a booking for the lecture of " . $lecture["name"] . ". The booking has been successfull");
```

Comment:
Mention in the mail if student is in waiting list
```suggestion
			//I send a confirmation email
			$mail_subject = "Confirmation of " . $lecture["name"] . " lecture booking";
			$mail_body = "You succesfully booked for the lecture of " . $lecture["name"] .". ($in_wait_list ? " The room is currently at full capacity, you have been placed in a waiting list." : "");
			$mail_result = mail($user_data["email"], $mail_subject, $mail_body);
```

Reply:
good idea